app/views/openid_connect/logout/index.html.erb

```diff
@@ -0,0 +1,24 @@
+<% title t('openid_connect.logout.heading', app_name: APP_NAME) %>
+
+<div class='text-center'>
+  <%= image_tag(asset_url('user-access.svg'), width: '280', alt: '') %>
```

Comment:
Can we assign a `height` here to prevent page relayout?

Related:

- https://www.smashingmagazine.com/2020/03/setting-height-width-images-important-again/
- https://www.aleksandrhovhannisyan.com/blog/setting-width-and-height-on-images/

Reply:
I set the height to 91 based on this math and the size of the image intrinsically: (280/360)*117 = 91

![image](https://user-images.githubusercontent.com/1430443/192813748-8ab70ee4-33fb-4ee5-aa69-12d6fd6b7dc4.png)

If that is the wrong way to do it, let me know 😀

Reply:
Forgot to add, it was copied from https://github.com/18F/identity-idp/blob/c12f697e0958370e85bee701425ce84743878ffb/app/views/users/authorization_confirmation/new.html.erb#L5

It's also used on https://github.com/18F/identity-idp/blob/c12f697e0958370e85bee701425ce84743878ffb/app/views/devise/sessions/new.html.erb#L7

I can also update it in those places?

Reply:
Yep, I think as long as you maintain the original aspect ratio, the math checks out 👍  (Alternatively, we could choose to resize the original image to avoid math and _maybe_ avoid browser scaling distortion, but not critical unless it looks obviously distorted)

Reply:
Good find on the other pages! I think we should update them, yeah.

Reply:
(Also, while looking at it, I think the `alt` might warrant a rethink as well, since it's probably a stretch to call this purely "decorative")

Reply:
Cool, updated in 001be27.  It doesn't look distorted:
![image](https://user-images.githubusercontent.com/1430443/192819010-71e04222-8f7c-4f76-bd04-b718cf6c0e19.png)